### PR TITLE
MDEXP-125, MDEXP-126: Adding rules file for series field and physical description

### PR DESCRIPTION
**Pull request by KVupp**
https://issues.folio.org/browse/MDEXP-125
https://issues.folio.org/browse/MDEXP-126

## Purpose
- Mapping series field
    --  490 field with first indicator set to 0 and second indicator blank.
- Mapping Physical description field to 300
   -- 300 field with both indicators left blank

ADDITIONAL CHANGES
- corrected the log4j2.properties having an incorrect format(it was named log4j2, but was using log4j format)
## Approach
- Add to rulesDefault file
- Check if multiple entries of series are handled 
- Check if multiple entries of physical description are handled 

## TODO
[] Check if we can write negative test cases
[] Check if validation can be done on the marc record generated

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by KVupp** [comment]
Verification of the example field
Screen Shot 2020-04-26 at 11 29 32 PM

File: src/main/resources/rules/rulesDefault.json [hunk 1]
```diff
@@ -49,18 +58,65 @@
...
+            "value": "0"
+          }
+        }
+      },
+      {
+        "indicator": "2",
+        "translation": {
+          "function": "set_value",
+          "parameters": {
+            "value": " "
```
Comment (Igor-Gorchakov) [inline src/main/resources/rules/rulesDefault.json:89]: it is blank by default

Comment (KVupp) [reply src/main/resources/rules/rulesDefault.json:89]: actually it doesn't, while reading it checks for the charAt(0) which throws a NPE if it is empty
https://github.com/folio-org/mod-data-export/blob/a19fc78b6d8be3b61053dd085cd631eaf29ecf8b/src/main/java/org/folio/service/mapping/writer/impl/AbstractRecordWriter.java#L69

Comment (Igor-Gorchakov) [reply src/main/resources/rules/rulesDefault.json:89]: I mean - no need to specify dataSource for ind2, it is blank already if nothing specified 
https://github.com/folio-org/mod-data-export/blob/master/src/main/java/org/folio/service/mapping/writer/fields/RecordDataField.java#L14

Comment (KVupp) [reply src/main/resources/rules/rulesDefault.json:89]: That will not work either, 300 field was generated as below if datasources are removed. Anyway having it explicitly that it is blank may be more readable. We can change it going forward if needed


300 | __ | $axx, 238 p. : ill. ; 24 cm.$axxix, 224 Seiten Illustrationen

**Pull request merged**